exporter/skywalkingexporter/skywalking_benchmark_test.go

```diff
@@ -145,7 +145,7 @@ func doInit(numStream int, t *testing.T) (*swExporter, *grpc.Server, *mockLogHan
 		},
 	}
 
-	oce := newExporter(context.Background(), tt, componenttest.NewNopTelemetrySettings())
+	oce := newLogsExporter(context.Background(), tt, componenttest.NewNopTelemetrySettings())
```

Comment:
Please ensure that tech debt is addressed before adding new features: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/6116

Reply:
I will fix it in another pr.